bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
+    # elif "team_join" == response["type"]:
+    #     if NEWUSERGREETING:
+    #         new_user_message(response)
+    #     USER_LIST = sc.api_call(  # noqa: 841
+    #         "users.list",
+    #     )
+
+    log(response)   
+
+def log(response):
+    global engine
+    def add_user(r):
+        q = engine.execute("SELECT last_name FROM users WHERE slack_id=%s", r["user"])
```

Comment:
I'm sorry, I must not have been clear enough. Let me clarify:

You should use SQL queries for testing out schema and getting a feel for SQL in general.
When that goes to code you should avoid using hard coded queries. Hard coding a query is when you put actual SQL into your code, this should be avoided except when absolutely necessary. 

I can double check this for you if you need me to, but in most llibraries like SQLAlch you should be able to make your query without using SQL. This is a separate feature from the ORM, please have a look around and see if you can find how to do it on your own, if you get stuck feel free to reach out and i'll help you look for it.